server/src/main/resources/static/old-console/js/rules-0.0.2.js

```diff
@@ -114,6 +114,7 @@ function makeLoadByInterval(rule) {
 function makeLoadByPeriod(rule) {
   var retVal = "";
   retVal += "<span class='rule_label'>period</span><input type='text' name='period' " + "value='" + rule.period + "'/>";
+  retVal += "<span class='rule_label'>includeFuture</span><input type='text' name='includeFuture' " + "value='" + true + "'/>";
```

Comment:
This looks a legacy UI. Do you have a plan to update the recent UI as well?

Reply:
Yes, I'll open a PR in [druid-console](https://github.com/druid-io/druid-console) to update the UI.